File: backend/SmartSusChef.Api/Models/SalesData.cs
namespace SmartSusChef.Api.Models;

public class SalesData
{
    public Guid Id { get; set; }
    public DateTime Date { get; set; }
    public Guid RecipeId { get; set; }
    public int Quantity { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }

    // Navigation properties
    public Recipe Recipe { get; set; } = null!;
}

File: mobile/app/src/main/java/com/smartsuschef/mobile/data/models/SalesData.kt
package com.smartsuschef.mobile.data.models

import java.util.Date

data class SalesData(
    val id: String, // Guid (.NET) --> String in Kotlin
    val storeId: Int,
    val date: String, // DateTime (.NET) -> String (ISO format) in Kotlin
    val recipeId: String,
    val quantity: Int,
    val createdBy: String,  // To record the User who entered the data
    val createdAt: String, // DateTime (.NET) -> String (ISO format) in Kotlin
    val recipeName: String? = null,

    // Edit history tracking (for audit trail)
    val editHistory: List<EditHistory>? = null

)